kubernetes_state/test_kubernetes_state.py

```diff
@@ -64,6 +69,18 @@ class TestKubernetesState(AgentCheckTest):
         NAMESPACE + '.container.waiting',
     ]
 
+    ZERO_METRICS_V040 = [
```

Comment:
If  ZERO_METRICS is not used, could we remove it so it's cleaner ?

Reply:
Yes, this was too much of a copy from others work.